server/src/ClientToDBAPI/InvestigationInfo/mainRoute.ts

```diff
@@ -0,0 +1,13 @@
+import { Router, Request, Response } from 'express';
+
+import { graphqlRequest, graphqlURL, baseUrl } from '../../GrapqlHTTPRequest';
+import { GET_INVESTIGATION_INFO } from '../../DBService/InvestigationInfo/Query';
+
+const investigationInfo = Router();
+
+investigationInfo.get('/staticInfo', (request: Request, response: Response) => {
+    graphqlRequest(baseUrl+process.env.PORT+graphqlURL, GET_INVESTIGATION_INFO)
```

Comment:
לא אמור להיות פרמטר של איזה מידע להביא? id?

Reply:
Make a function that only receives the query and the variables, the GraphQL url will be constant